### ISSUE TITLE
Expired intermediate certificate in the chain should be an error

### ISSUE THREAD

**Issue by taxilian** [ssllabs/ssllabs-scan#860]
Hi,

I love your service! Super handy, I use it every time I suspect an issue. Recently (an hour ago =]) I had an issue which was only showing up on some systems, so the first thing I checked was an ssllabs scan, which reported "A+" -- but it turns out I had a just-barely-expired letsencrypt intermediate CA certificate in the chain.

Seems like that is something which should prevent an A+ rating, no?

Thanks for the tool! Just hoping to help make it even better,

Richard

**Comment by wordreference** [comment]
Yes, my letsencrypt certificate passes here and the section for Android 6 & 7 is green, but the fact is that the certificate doesn't work for many Android 6 & 7 users. They get a security warning when visiting my website. If coming through an app, it silently fails.

In short, thanks for the great tool, but it is a shame that it doesn't warn us about Android 6 & 7 problems w/letsencrypt.

I am happy to provide more details if you are interested.

**Comment by ArchangeGabriel** [comment]
Unfortunately Android in SSL Labs has never been about Android itself, but rather Chromium as the default Android browser. What is available to apps as a SSL stack is singularly different.
